index.js

```diff
@@ -27,7 +27,12 @@ module.exports = class Kernal {
   // handles running code.
   static codeHandler (code, ethInterface = new Interface(new Environment())) {
     const instance = Wasm.instantiateModule(code, {
-      'ethereum': ethInterface.exportTable
+      'ethereum': ethInterface.exportTable,
+      'debug': {
+        'print': function (offset, length) {
+           console.log(new Uint8Array(instance.exports.memory, offset, length).toString())
```

Comment:
Well actually this won't have `instance` at that time. Let me think of a cleaner interface.